test/core/QCheck_expect_test.expected

```diff
@@ -498,21 +501,21 @@ l=[1], fold_left=1, fold_right=0
 
 --- Failure --------------------------------------------------------------------
 
-Test fold_left fold_right uncurried failed (97 shrink steps):
+Test fold_left fold_right uncurried failed (44 shrink steps):
 
-({(1, 7) -> 0; _ -> 7}, 0, [1; 0])
+({(0, 7) -> 1; _ -> 0}, 0, [7])
 
 --- Failure --------------------------------------------------------------------
 
-Test fold_left fold_right uncurried fun last failed (21 shrink steps):
+Test fold_left fold_right uncurried fun last failed (34 shrink steps):
 
-(0, [1], {(0, 1) -> 0; _ -> 1})
+(0, [1], {(1, 0) -> 1; _ -> 0})
 
 --- Failure --------------------------------------------------------------------
 
-Test fold_left test, fun first failed (40 shrink steps):
+Test fold_left test, fun first failed (275 shrink steps):
```

Comment:
hm

Reply:
Yes, this one looks worse indeed.
Recall that this is only part of the story, as these printed numbers count only *successful shrink attempts*.
For a more complete story, compare the `fold_left test, fun first` lines (final lines) in the tables here: https://github.com/c-cube/qcheck/pull/235#issuecomment-1103781517
The Q1 columns labelled `#succ/#att` report counts of both *successful* and *total* attempts.
Admittedly, even there for seed 1234 both numbers go up...

Reply:
OK, I dug a bit.
Turns out this was caused by a linear-time `char` shrinker that was trying to reduce a one-character string one ASCII-step at a time. I've thus replaced it by a faster shrinker that
- uses the underlying `int` shrinker
- shrinks towards `'a'` (like QCheck2)

This brings the *successful* shrink count down to 36 :smile: